packages/next/taskfile-swc.js

```diff
@@ -0,0 +1,91 @@
+// taskr babel plugin with Babel 7 support
+// https://github.com/lukeed/taskr/pull/305
+
+const path = require('path')
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const transform = require('@swc/core').transform
+
+module.exports = function (task) {
+  // eslint-disable-next-line require-yield
+  task.plugin('swc', {}, function* (
+    file,
+    serverOrClient,
+    { stripExtension, dev } = {}
+  ) {
+    // Don't compile .d.ts
+    if (file.base.endsWith('.d.ts')) return
```

Comment:
It may be possible to remove this check and change the glob pattern with the exception of d.ts?